src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -280,6 +291,31 @@ private async void DiagnosticService_DiagnosticsUpdated(object sender, Diagnosti
             }
         }
 
+#pragma warning disable VSTHRD100 // Avoid async void methods
+        private async void RequestExecutionQueue_Errored(object sender, RequestShutdownEventArgs e)
+#pragma warning restore VSTHRD100 // Avoid async void methods
```

Comment:
❗ Do not suppress VSTHRD100. This method must either be `async Task`, or just `void`.

Reply:
Am I exempt because there already was one of these in this file? 😛

Reply:
If you just moved existing code here then yes (i.e. this is the same suppression, just in a different location than it was previously).

Reply:
I submitted #47115 to remove the other one.

Reply:
Thank you, I'll copy that pattern once it merges through :)